booking.py

```diff
@@ -7,6 +7,8 @@
 from .utils import Count
 from .utils import Histogram
 
+from ROOT import gROOT
+gROOT.SetBatch(True)
```

Comment:
what does this do?

Reply:
this puts ROOT in batch mode, e.g., it's not firing up a gui thread

Reply:
aaah ok thanks